lib/engines/hcktest/tests.rb

```diff
@@ -349,9 +378,41 @@ def create_project_package
       @project.result_uploader.upload_file(res['hostprojectpackagepath'], r_name)
     end
 
+    def build_system_info(info)
+      @guest_systems_info ||= {}
+
+      system_info = {
+        'Host' => info['Host Name'],
+        'OS' => "#{info['OS Name']} #{info['OS Version']}",
+        'System' => "#{info['System Manufacturer']} #{info['System Model']} #{info['System Type']}",
+        'CPU' => info['Processor(s)'].join(' '),
+        'Memory' => info['Total Physical Memory'],
+        'BIOS' => info['BIOS Version']
+      }
+
+      @guest_systems_info[info['Host Name']] = system_info
+    end
+
+    def load_clients_system_info
+      parser = SysInfoParser.new
+
+      client_sysinfo = parser.parse(@tools.get_machine_system_info(@client.name))
+      build_system_info(client_sysinfo)
+
+      return if @support.nil?
+
+      support_sysinfo = parser.parse(@tools.get_machine_system_info(@support.name))
+      build_system_info(support_sysinfo)
+    end
+
```

Comment:
The store variable is named `guest_systems_info` but the method name says `clients_system_info`. Keep them consistent.

Reply:
fixed